src/storage/v3/indices.cpp

```diff
@@ -278,9 +284,8 @@ bool LabelIndex::CreateIndex(LabelId label, VerticesSkipList::Accessor vertices)
   }
   try {
     auto acc = it->second.access();
-    for (auto &lgo_vertex : vertices) {
-      auto &vertex = lgo_vertex.vertex;
-      if (vertex.deleted || !utils::Contains(vertex.labels, label)) {
+    for ([[maybe_unused]] auto &vertex : vertices) {
```

Comment:
Why `[[maybe_unused]]`?

Reply:
I forgot to remove it, before it was destructured into pk and vertex_data